### ISSUE TITLE
Can not QAT on multiple GPUs with DDP

### ISSUE THREAD

**Issue by jia0511** [Xilinx/Vitis-AI#1084]
Hello xilinx,
 I encountered the following problem when I quantified the Resnet50 model through dual-card QAT, and the quantified full days are as attached.

#########9                                        | 36/181 [00:00<00:00, 4252.42it/s, OpInfo: name = DistributedDataParallel/ResNet[module]/Sequential[layer1]/Bottleneck[2]/BatchNorm2d[bn3]/8690, type = batch_norm]   
[VAIQ_ERROR]: Unsupported Ops: {'_record_function_enter'}

[error.txt](https://github.com/Xilinx/Vitis-AI/files/9947819/error.txt)
